### PR TITLE
fix(web/applications): fixed issue with slashed in URL builder helper

### PR DESCRIPTION
**Pull request by francescobrenci-AR**
## Describe your changes

fixed issue with slashed in URL builder NJK helper

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/src/server/views/applications/dashboard.njk [hunk 1]
```diff
@@ -32,7 +32,7 @@
 				<div class="pins-dashboard-box">
 					<h3 class="govuk-heading-l pins-dashboard-box--title">Create new case</h3>
 					<a class="govuk-button govuk-button--secondary pins-dashboard-box--btn"
-					   href="{{ 'case-create'|url }}">Create new case here</a>
+					   href="{{ 'base-url'|url }}/create-new-case">Create new case here</a>
```
Comment (lutamKC) [inline apps/web/src/server/views/applications/dashboard.njk:35]: this looks like a fast fix. 
I'm sure this problem of the double / comes from a recent fix and probably affects case-view as well. might be better to investigate what happened

**Review by lutamKC — APPROVED** [review]

File: apps/web/src/server/views/applications/dashboard.njk (hunk 1 above)
Comment (francescobrenci-AR) [reply apps/web/src/server/views/applications/dashboard.njk:35]: we do need away to build urls outside of those case and `base-url` helps with that.
For now I'm addressing only the double slash in that specific URL

**Pull request merged**